turborepo-tests/integration/tests/find_turbo/hoisted.t

```diff
@@ -4,24 +4,22 @@ Setup
 
 Make sure we use local and do not pass --skip-infer to old binary
   $ ${TESTDIR}/set_version.sh $(pwd) "1.0.0"
-  $ ${TURBO} build --filter foo -vv
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Global turbo version: .* (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Repository Root: .*/hoisted.t (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Local turbo path: .*/hoisted.t/node_modules/turbo-(darwin|linux|windows)-(64|arm64)/bin/turbo (re)
-  [-0-9:.TWZ+]+ \[DEBUG] turborepo_lib::shim: Local turbo version: 1.0.0 (re)
```

Comment:
I think this is actually the line we want to check and the rest we can ignore aside from the last line where we verify that `--skip-infer` wasn't passed.

Reply:
Don't we care about the path that's used also?